docs/executor_tutorial/google_lifesciences.rst

```diff
@@ -0,0 +1,880 @@
+        output: snakemake-testing-data/calls/all.vcf
+        jobid: 2
+        resources: mem_mb=15360, disk_mb=128000
+
+    Get status with:
+    gcloud config set project snakemake-testing
+    gcloud beta lifesciences operations describe projects/snakemake-testing/locations/us-west2/operations/622600526583374352
+    gcloud beta lifesciences operations list
+    [Fri Apr 17 20:49:57 2020]
+    Finished job 2.
+    7 of 9 steps (78%) done
+
+    [Fri Apr 17 20:49:57 2020]
+    rule plot_quals:
+        input: snakemake-testing-data/calls/all.vcf
+        output: snakemake-testing-data/plots/quals.svg
+        jobid: 1
+        resources: mem_mb=15360, disk_mb=128000
+
+    Get status with:
+    gcloud config set project snakemake-testing
+    gcloud beta lifesciences operations describe projects/snakemake-testing/locations/us-west2/operations/9350722561866518561
+    gcloud beta lifesciences operations list
+    [Fri Apr 17 20:53:10 2020]
+    Finished job 1.
+    8 of 9 steps (89%) done
+
+    [Fri Apr 17 20:53:10 2020]
+    localrule all:
+        input: snakemake-testing-data/plots/quals.svg
+        jobid: 0
+        resources: mem_mb=15360, disk_mb=128000
+
+    Downloading from remote: snakemake-testing-data/plots/quals.svg
+    Finished download.
+    [Fri Apr 17 20:53:10 2020]
+    Finished job 0.
+    9 of 9 steps (100%) done
+    Complete log: /home/vanessa/snakemake-work/tutorial/.snakemake/log/2020-04-17T202749.218820.snakemake.log
+
+
+We've finished the run, great! Let's inspect our results.
+
+Step 4: View Results
+::::::::::::::::::::
+
+The entirety of the log that was printed to the terminal will be available
+on your local machine where you submit the job in the hidden `.snakemake`
+folder under "log" and timestamped accordingly. If you look at the last line
+in the output above, you'll see the full path to this file.
+
+You also might notice a line about downloading results:
+
+.. code:: console
+
+    Downloading from remote: snakemake-testing-data/plots/quals.svg
+
+
+Since we defined this to be the target of our run
+
+.. code:: console
+
+
+    rule all:
+        input:
+            "plots/quals.svg"
+
+
+this fill is downloaded to our host too. Actually, you'll notice
+that paths in storage are mirrored on your filesystem (this is what the workers
+do too):
+
+
+.. code:: console
+
+    $ tree snakemake-testing-data/
+    snakemake-testing-data/
+    └── plots
+        └── quals.svg
+
+
+We can see the result of our run, quals.svg, below:
+
+.. image:: workflow/quals.svg
+
+
+And if we look at storage, we see that the result file (under plots) and intermediate
+results (under sorted_reads and calls) are saved there too!
+
+.. image:: workflow/results-google-storage.png
+
+The source folder contains a cache folder with archives that contain your working directories,
```

Comment:
One for each step? Not sure I understand this. Isn't it always the same workdir and didn't we want to upload it only once?

Reply:
The name is based on a hash of the contents, so if the working directory isn't changed it would just be one. I was referring to (generally) a cache folder for all the working directories that you produce across many snakemake runs. This is probably confusing so I'll reword it.

Reply:
Take a look and let me know if this reads better now.